core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -204,6 +204,8 @@ CudaUVMSpace::CudaUVMSpace() : m_device(Kokkos::Cuda().cuda_device()) {}
 
 CudaHostPinnedSpace::CudaHostPinnedSpace() {}
 
+int memory_threshold_g = 40000; // 5 kB
```

Comment:
was this supposed to be 5kB or 40? otherwise its fine.

Reply:
Typo, the comment was supposed to be 40